index.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-env node */
+"use strict";
+
+var fluid = require("infusion");
+// Make sure we get a good copy of gpii-express
+fluid.require("gpii-express");
+
+
+require(__dirname + "/src/js/server/devpmt-express.js");
```

Comment:
This formality seems unnecessary since require supports relative paths as in require("./src/js/server/devpmt-express.js") as we use elsewhere

Reply:
__dirname replaced with `./`